test/tools/fixtures/process_resource_script.in.js

```diff
@@ -128,10 +127,10 @@ async function main() {
 }
 
 main()
-  .then(() => {})
+  .then(() => null)
   .catch(e => {
     log({ error: { message: e.message, stack: e.stack, resources: getNewResources() } });
-    process.exit(1);
+    process.exit(2);
```

Comment:
Is exit code 2 needed anywhere?

Reply:
No, just changed this for debugging, "2" or any other number makes it obvious I hit our intentional exit point, when I was getting "undefined: undefined" as an error it was hard to tell where the script was crashing.